backend/src/Service/Subscription/SubscriptionService.php

```diff
@@ -188,23 +188,22 @@ public function updateSubscribeState($id, $status): string
         return $this->subscriptionManager->updateSubscribeState($id, $status);
     }
 
-    // reduce the number of available orders, call it only when create order success.
-
     /**
      * @param $storeOwner
      * @return string
      */
     public function updateRemainingOrders($storeOwner): string
```

Comment:
Functions like this one (which updates remaining orders by decreasing the field value by 1) why we name them explicitly according to their aims. I mean we can name the function like this: decreaseRemainingOrdersByOne, for example.